arithmetization/src/test/java/net/consensys/linea/ZkCounterTest.java

```diff
@@ -360,6 +367,8 @@ void modexpCall(boolean base, boolean exp, boolean mod) {
     assertEquals((!base && !exp && !mod) ? 0 : Integer.MAX_VALUE, lineCountMap.get(MODEXP));
     assertEquals(0, lineCountMap.get(RIP));
     assertEquals(0, lineCountMap.get(BLAKE));
+    assertEquals(0, lineCountMap.get("BLS"));
+    assertEquals(0, lineCountMap.get("POINT_EVAL"));
```

Comment:
Splitting hairs here, but it would be more logical to have `POINT_EVAL` precede `BLS`.